Don't put postgres data into home

dasJ opened the pull request.

dasJ committed:
Don't put postgres data into home

SlothOfAnarchy commented on tasks/configure.yml:
```diff
@@ -42,8 +42,8 @@
     - /etc/profile.d/postgresql.sh
-    - "{{ postgres_data_directory }}/.profile"
-    - "{{ postgres_data_directory }}/.bashrc"
+    - "/var/lib/postgresHome/.profile"
```
Please use a var for this, you can place it in the defaults.

Scriptkiddi committed:
Adds postgres_home_directory to allow variable paths

Replaces all occurences of /var/lib/postgresHome with the variable
Also adds the default var postgres_home_directory with the value
/var/lib/postgresHome

Commit:
Merge branch 'master' into dataHome

Scriptkiddi commented:
pls rereview

dasJ commented:
@Scriptkiddi Thanks for introducing an undocumented variable
